draft-irtf-cfrg-frost.md

```diff
@@ -157,6 +157,7 @@ The following notation and terminology are used throughout this document.
 * `NUM_SIGNERS` denotes the number of signers that participate in an invocation of FROST signing, where
   THRESHOLD_LIMIT <= NUM_SIGNERS <= MAX_SIGNERS.
   This value MUST NOT exceed p.
+* Each participant is represented by an identifier that is an integer in the range between 1 and NUM_SIGNERS. 
```

Comment:
I would move this to the start of section 5, where we first start talking about participants (outside of this terminology list).

Reply:
I'm cool with it in either location

Reply:
Done